tests/test_mod.py

```diff
@@ -287,25 +325,30 @@ def test_footprints_s1():
         datetime(2014, 10, 10), datetime(2014, 12, 31), producttype="GRD"
     )
 
+    footprints = api.to_geojson(products)
+    for footprint in footprints['features']:
+        print(footprint)
+        validation = geojson.is_valid(footprint['geometry'])
+        assert validation['valid'] == 'yes', validation['message']
+
     with open('tests/expected_search_footprints_s1.geojson', 'r') as geojson_file:
         expected_footprints = geojson.loads(geojson_file.read())
-        # to compare unordered lists (JSON objects) they need to be sorted or changed to sets
-        assert set(api.to_geojson(products)) == set(expected_footprints)
+    # to compare unordered lists (JSON objects) they need to be sorted or changed to sets
+    assert set(footprints) == set(expected_footprints)
 
 
-@my_vcr.use_cassette
 @pytest.mark.scihub
-def test_footprints_s2():
-    api = SentinelAPI(**_api_auth)
-    products = api.query(
-        get_coordinates('tests/map.geojson'),
-        "20151219", "20151228", platformname="Sentinel-2"
-    )
+def test_footprints_s2(products):
+    footprints = SentinelAPI.to_geojson(products)
+    for footprint in footprints['features']:
+        print(footprint)
```

Comment:
I don't think printing messages NOT from pytest is a good idea in this context - I think this might catch future contributors by surprise.